Sources/SwiftProtobuf/JSONDecodingOptions.swift

```diff
@@ -25,5 +25,9 @@ public struct JSONDecodingOptions {
   /// ignored, an error will be raised if one is encountered.
   public var ignoreUnknownFields: Bool = false
 
+  /// If unknown enum cases in the JSON should be defaulted to initial enum case.
+  /// If they aren't ignored, an error will be raised if one is encountered.
+  public var ignoreUnknownEnumValues: Bool = false
+
```

Comment:
According to protocolbuffers/protobuf#7392 the C++ implementation has a single option "ignoreUnknownFields" that also causes unknown enum values to be ignored.  I'd prefer to use that here:  It keeps us aligned with the C++ implementation and also reduces the matrix of options we need to test.

Reply:
I agree, separated option specific for enums is redundant. I can hardly imagine somebody will prefer to only ignore enums but fall on other fields. Done.